_schedule/talks/2016-07-18-09-00-keynote-1.md

```diff
@@ -4,20 +4,21 @@ category: talk
 date: 2016-07-18 09:00
 layout: session-details
 permalink: /talks/keynote-1/
-presenters: Saron Yitbarek, CodeNewbie
-photo_url: https://placem.at/people?w=400&h=400&random=3
 published: true
 sitemap: false
 room: Auditorium (G-06)
 summary: ''
 title: Keynote
 track: ''
+presenters:
```

Comment:
@FlipperPA this reflects the changes we'll need to make in the data exporter. This allows us to have multiple presenters which we can add post-export by hand.